pages/learn/deploy/delta.md

```diff
@@ -26,6 +26,14 @@ Before devices can update using deltas, a delta must be generated between the re
 
 On-demand deltas may take a while to generate, depending primarily on the size of the images involved. While the delta is generating, the device keeps polling the API, checking if the delta is ready. If a delta already exists, this step is skipped.
 
+if the delta is being generated, and the supervisor tries to download it, you will get this line in your supervisor log file:
+
+'''
+Failed to download application 'registry2.resin.io/foo/bar' due to 'Delta server is still processing the delta, will retry
```

Comment:
> Failed to download application 'registry2.resin.io/foo/bar' due to 'Delta server is still processing the delta, will retry

Are you sure logs print this message?

Reply:
@dfunckt , I was following the issue attached together with some comment in this Front thread https://app.frontapp.com/open/cnv_l1pu4x
But you are right, I'm not sure if this is really printed, and if it is if this is the cause.